packages/core/getBusyTimes.ts

```diff
@@ -144,7 +144,7 @@ export async function getBusyTimes(params: {
   const bookingSeatCountMap: { [x: string]: number } = {};
   const busyTimes = bookings.reduce(
     (aggregate: EventBusyDetails[], { id, startTime, endTime, eventType, title, ...rest }) => {
-      if (rest._count?.seatsReferences) {
+      if (eventType?.seatsPerTimeSlot) {
```

Comment:
In unit tests the prisma mock doesn't return `seatsReferences` which ends up failing the tests.